[2014-08-07 12:04:23] log.ERROR: 500 - Exception @ /user/login
exception 'Illuminate\Session\TokenMismatchException' in /home/htdocs/turnos1/app/filters.php:99
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array)
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request), Array)
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'csrf', Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'postLogin')
#5 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#8 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#9 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#13 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#14 {main} [] []
[2014-08-07 12:04:34] log.ERROR: 500 - Exception @ /user/login
exception 'Illuminate\Session\TokenMismatchException' in /home/htdocs/turnos1/app/filters.php:99
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(159): call_user_func_array(Object(Closure), Array)
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(262): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request), Array)
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(185): Illuminate\Routing\Controllers\Controller->callFilter(Object(Illuminate\Routing\Route), 'csrf', Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(109): Illuminate\Routing\Controllers\Controller->callBeforeFilters(Object(Illuminate\Routing\Router), 'postLogin')
#5 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postLogin', Array)
#6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#7 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#8 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#9 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#13 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#14 {main} [] []
[2014-08-07 21:00:32] log.ERROR: 404 - Exception @ /assets/js/jquery.min.map
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:30:08] log.ERROR: 404 - Exception @ /turns/asset/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:30:09] log.ERROR: 404 - Exception @ /turns/asset/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:30:50] log.ERROR: 404 - Exception @ /turns/asset/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:30:51] log.ERROR: 404 - Exception @ /turns/asset/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:31:43] log.ERROR: 404 - Exception @ /turns/asset/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:32:05] log.ERROR: 404 - Exception @ /turns/assets/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:32:11] log.ERROR: 404 - Exception @ /turns/assets/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
[2014-08-07 21:32:44] log.ERROR: 404 - Exception @ /turns/assets/js/locales/bootstrap-datetimepicker.es.js
exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(574): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/htdocs/turnos1/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(550): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/htdocs/turnos1/public/index.php(49): Illuminate\Foundation\Application->run()
#5 /home/htdocs/turnos1/server.php(19): require_once('/home/htdocs/tu...')
#6 {main} [] []
